pyproject.toml

```diff
@@ -22,9 +22,11 @@ html5lib = "^1.1"
 flask-expects-json = "^1.7.0"
 celery = "^5.2.7"
 redis = "^4.5.1"
+pydantic = "^1.10.4"
+flask-pydantic = "^0.11.0"
```

Comment:
Do we need to install both differently?

Reply:
No, actually I forgot to remove it. will do that.

Reply:
Done.